event/src/app/api/events/[id]/route.ts

```diff
@@ -0,0 +1,69 @@
+import { IUserPayload } from "@/app/api/users/login/route";
+import { NextRequest, NextResponse } from "next/server";
+import * as jose from 'jose'
+import Event, { Events } from "@/models/Event";
+import dbConnect from "@/lib/mongo";
+import { EventSchema, IEvent } from "../host/route";
+
+export async function DELETE(req: NextRequest, content: any) {
```

Comment:
Suggest to destructure params and add the types for type checking

```suggestion
export async function DELETE(req: NextRequest, { params }: { params: { id: string } }) {
```